scripts/yg-privacyenhanced.py

```diff
@@ -21,7 +21,29 @@ class YieldGeneratorPrivacyEnhanced(YieldGeneratorBasic):
 
     def __init__(self, wallet_service, offerconfig):
         super().__init__(wallet_service, offerconfig)
-
+        
+    def select_input_mixdepth(self, available, offer, amount):
```

Comment:
It's not the fault of this PR, as it's the same in the base class, but the argument `offer` is unused.

Reply:
Yep but I don't think I can remove it ?

From what I understand `select_input_mixdepth` is called in `oid_to_order` in the base class here: https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/09c041e40940d327984d51632ffa2e56d6be5870/jmclient/jmclient/yieldgenerator.py#L115-L128

So unless I overwrite `oid_to_order` too, python will complain about a missing argument if I am not missing something.

Reply:
Yeah needs fixing separately.